ports/rtabmap/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "rtabmap",
-  "version-string": "0.20.3",
-  "port-version": 1,
+  "version-string": "0.20.8",
```

```suggestion
  "version": "0.20.8",
```
This might be `version`, please see doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions.